src/components/OrganizationListGridItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Organization } from '@pdc/sdk';
+import './OrganizationListGridItem.css';
+
+interface OrganizationListGridItemProps {
+	organization: Organization;
+	active?: boolean;
+}
+
+export const OrganizationListGridItem = ({
+	organization,
+	active = false,
+}: OrganizationListGridItemProps) => (
+	<Link
+		to={`/organizations/${organization.id}`}
+		className={`organization-list-grid-item ${active ? 'active' : ''}`.trim()}
+	>
+		<div className="orginization-name">{organization.name}</div>
```

Comment:
```suggestion
		<div className="organization-name">{organization.name}</div>
```

(Will need to update CSS accordingly.)